.rubocop.yml

```diff
@@ -42,6 +38,7 @@ Naming/FileName:
   Description: Some files violates the snake_case convention
   Exclude:
     - 'lib/beaker-docker.rb'
+    - Gemfile.local
```

Comment:
in the future we should really get rid of this file and just add beaker-rspec add development dependency in the gemspec

Reply:
Or merge it into `Gemfile`: https://github.com/voxpupuli/beaker-docker/pull/104